apps/docs/content/guides/platform/manage-your-usage/disk-throughput.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: 'manage-usage-disk-throughput'
+title: 'Manage Disk Throughput usage'
+---
+
+## What you are charged for
+Each database has a dedicated disk. You are charged for the provisioned disk throughput. You can configure disk throughput in the advanced settings on the [project's Compute and Disk page](https://supabase.com/dashboard/project/_/settings/compute-and-disk).
```

Comment:
Maybe add a sentence like: Unless you specifically opt into additional provisioned disk throughput, you will not be charged for it.